utils/unittest/googletest/include/gtest/gtest-printers.h

```diff
@@ -683,6 +683,10 @@ void PrintTo(const ::std::pair<T1, T2>& value, ::std::ostream* os) {
   *os << ')';
 }
 
+#if GTEST_LANG_CXX11
+inline void PrintTo(std::nullptr_t, ::std::ostream* os) { *os << "(nullptr)"; }
+#endif  // GTEST_LANG_CXX11
+
```

Comment:
Can you explain why this is necessary?

Reply:
More detail about this issue https://github.com/google/googletest/issues/1788